01-tamagotchi/src/lib.rs

```diff
@@ -1,18 +1,71 @@
 #![no_std]
 
-use gstd::prelude::*;
+use tmg1_io::*;
+use gstd::{exec, msg, debug, prelude::*};
+
+static mut NAME: Option<String> = None;
+static mut AGE: Option<u64> = None;
```

Comment:
You are to use `Tamagotchi` type here.
```suggestion
static mut STATE: Option<Tamagotchi> = None;
```